modules/plugin/imagemosaic/src/test/java/org/geotools/gce/imagemosaic/ImageMosaicReaderTest.java

```diff
@@ -5038,6 +5042,41 @@ public void testOverviewSupportFiles() throws Exception {
 
     @Test
     public void testBandsSelection() throws Exception {
+        JAI.getDefaultInstance()
+                .setImagingListener(
+                        new ImagingListener() {
+                            final Logger LOGGER = Logging.getLogger("javax.media.jai");
+
+                            @Override
+                            public boolean errorOccurred(
+                                    String message,
+                                    Throwable thrown,
+                                    Object where,
+                                    boolean isRetryable)
+                                    throws RuntimeException {
+                                thrown.printStackTrace(System.out);
+                                return false;
```

Comment:
I'm not sure I have understood this part.
Is it redirecting the exception stack trace to system out? Is it intentional? (or something there for previous debugging?)